device/transport/mqtt/devdoc/mqtt_requirements.md

```diff
@@ -309,3 +319,13 @@ The `getTwinReceiver` method creates a `MqttTwinReceiver` object for the twin re
 **SRS_NODE_DEVICE_MQTT_16_055: [** `disableMethods` shall call its callback with no arguments when the `UNSUBACK` packet is received. **]**
 
 **SRS_NODE_DEVICE_MQTT_16_046: [** `disableMethods` shall call its callback with an `Error` if an error is received while unsubscribing. **]**
+
+### disableTwin
+
+**SRS_NODE_DEVICE_MQTT_16_062: [** `disableTwin` shall call its callback immediately if the MQTT connection is already disconnected. **]**
+
+**SRS_NODE_DEVICE_MQTT_16_063: [** `disableTwin` shall unsubscribe from the topics for twin messages. **]**
+
+**SRS_NODE_DEVICE_MQTT_16_064: [** `disableTwin` shall call its callback with no arguments when the `UNSUBACK` packet is received. **]**
```

Comment:
>`UNSUBACK` [](start = 100, length = 10)

If no UNSUBACK received?

Reply:
I feel it's the job of the mqtt library to come back either with unsuback or an error. if we have to introduce timeouts for this type of operations I want a cohesive approach, not one-offs on some specific APIs.

---
In reply to: [145747906](https://github.com/Azure/azure-iot-sdk-node/pull/111#discussion_r145747906) [](ancestors = 145747906)